Subject: Default to kdesu password remembering on, but remember last state
From: Lubos Lunak
Feature: bnc#386531
Patch-upstream: no
Relates: kdebase4/kdesu-remember-keep-password.diff, kdelibs3/kdesu-settings.diff

Index: kdesu/kdesu/kdesu.cpp
===================================================================
--- kdesu/kdesu/kdesu.cpp	(revision 810363)
+++ kdesu/kdesu/kdesu.cpp	(working copy)
@@ -382,6 +382,7 @@
             change_uid = false;
         password = dlg.password();
         keep = dlg.keep();
+        KConfigGroup(config,"Passwords").writeEntry("Keep", keep);
         data.setSilent( KStartupInfoData::No );
         KStartupInfo::sendChange( id, data );
     }
